### ISSUE TITLE
MD5 check for journal.fnt failed,

### ISSUE THREAD

**Issue by thunderpants** [Klius/readable-omikron#1]
I  made a copy of my Steam ,newly installed, version to E:downloads

I also installed Gog version to C:games

GUI log  for Gog version with x64 tool + x86 tool on win10 :

Selected /Games/Omikron - The Nomad Soul as Omikron dir
Checking /Games/Omikron - The Nomad Soul\FONTS
Directory /Games/Omikron - The Nomad Soul\FONTS is invalid
ERROR make sure that /Games/Omikron - The Nomad Soul is the correct directory

GUI log  for Steam version  with x64 tool + x86 tool on win10 :

Selected /Downloads/Omikron as Omikron dir
Checking /Downloads/Omikron\FONTS
/Downloads/Omikron\FONTS Exists!
Checking journal.fnt
MD5 check for journal.fnt failed, expected 68e730bf3e67b36dca7892a8dc4aa26c got e5a36e5fbe108168ff11f79e33a8e946
ERROR make sure that /Downloads/Omikron is the correct directory

Steam and Gog version files are almost the same apart from a few Gog specific files.

**Comment by Klius** [comment]
I'll look into it, and get back to you,
Thanks for the issue ticket!

**Comment by Klius** [comment]
@thunderpants I've disabled the MD5 check, can you try it again with the [latest release](https://github.com/Klius/readable-omikron/releases/tag/1.0.2)?

**Comment by thunderpants** [comment]
Steam version all ok:

Selected /Downloads/Omikron as Omikron dir
Checking /Downloads/Omikron\FONTS
/Downloads/Omikron\FONTS Exists!
Checking journal.fnt
All checks OK!
Renaming the fonts to be replaced
Renaming of sneak.fnt OK!
Renaming of menusave.fnt OK!
Renaming of menuintr.fnt OK!
Renaming of parchemi.fnt OK!
Renaming done!
Replacing fonts with journal.fnt
Replacing of sneak.fnt OK!
Replacing of menusave.fnt OK!
Replacing of menuintr.fnt OK!
Replacing of parchemi.fnt OK!
Replacing done!
Replacing of the font was a success, have fun!

Gog version:

Selected /Games/Omikron - The Nomad Soul as Omikron dir
Checking /Games/Omikron - The Nomad Soul\FONTS
Directory /Games/Omikron - The Nomad Soul\FONTS is invalid
ERROR make sure that /Games/Omikron - The Nomad Soul is the correct directory

So I copied gog game folder to E:downloads:

Selected /Downloads/Omikron - The Nomad Soul as Omikron dir
Checking /Downloads/Omikron - The Nomad Soul\FONTS
/Downloads/Omikron - The Nomad Soul\FONTS Exists!
Checking journal.fnt
All checks OK!
Renaming the fonts to be replaced
Renaming of sneak.fnt OK!
Renaming of menusave.fnt OK!
Renaming of menuintr.fnt OK!
Renaming of parchemi.fnt OK!
Renaming done!
Replacing fonts with journal.fnt
Replacing of sneak.fnt OK!
Replacing of menusave.fnt OK!
Replacing of menuintr.fnt OK!
Replacing of parchemi.fnt OK!
Replacing done!
Replacing of the font was a success, have fun!


Not sure why the gog one didn't work in C:games.

Anyway both versions are ok now.
I copied the Gog font folder back to C:games
Great job ,fonts in-game are nice and clear now ,well done !

P.S:
I had to restart my pc before the tool would open after downloading it from github.
Not sure if that was just an issue with my pc.
I know something similar can happen after installing a game.
